ta/digest_perf/include/ta_digest_perf.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2015, Linaro Limited
+ * All rights reserved.
+ */
+
+#ifndef TA_DIGEST_PERF_H
+#define TA_DIGEST_PERF_H
+
+#define TA_DIGEST_PERF_UUID { 0x614789f2, 0x39c0, 0x4ebf, \
+	{ 0xb2, 0x35, 0x92, 0xb3, 0x2a, 0xc1, 0x07, 0xed } }
```

Comment:
This change reuses a TA UUID but changes its interface (new param type for `TA_DIGEST_PERF_CMD_PROCESS`). This is not clean. I suggest to rather add a new command: `TA_DIGEST_PERF_CMD_PROCESS_MAC` to be used for MACs.

I am fine with the renaming of `ta_sha_perf` to `ta_disgest_perf`, but maybe not needed.

Reply:
Thank you for your suggestion.
I agree that it would be better to add a new command to be  used for MACs instead of changing the interface.
However, in this merge, SM3 algorithm is added.In my opinion, SM3 is not included in sha.So, I think  the renaming of ta_sha_perf to ta_disgest_perf is essential.